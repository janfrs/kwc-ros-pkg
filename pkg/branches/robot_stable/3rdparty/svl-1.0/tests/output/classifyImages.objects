<Object2dSequence>
    <Object2dFrame id="image001">
        <Object name="test" x="36" y="360" w="32" h="32" pr="0.648815" />
        <Object name="test" x="104" y="340" w="32" h="32" pr="0.51196" />
        <Object name="test" x="321.6" y="360" w="38.4" h="38.4" pr="0.696299" />
        <Object name="test" x="69.12" y="345.6" w="46.08" h="46.08" pr="0.618884" />
        <Object name="test" x="74.6496" y="223.949" w="66.3552" h="66.3552" pr="0.50656" />
        <Object name="test" x="481.579" y="326.786" w="137.594" h="137.594" pr="0.725381" />
        <Object name="test" x="41.2782" y="309.587" w="165.113" h="165.113" pr="0.65634" />
    </Object2dFrame>
</Object2dSequence>
