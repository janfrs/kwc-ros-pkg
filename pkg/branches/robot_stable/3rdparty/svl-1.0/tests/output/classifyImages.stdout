--- Running classifier on 1 images/sequences...
--- Running classifier on 2-channel image
